plugins/optimization-detective/detection.php

```diff
@@ -76,10 +76,15 @@ static function ( OD_URL_Metric_Group $group ): array {
 		$detect_args['urlMetricGroupCollection'] = $group_collection;
 	}
 
+	$detection_script_file = 'detect.min.js';
+	if ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) {
+		$detection_script_file = 'detect.js';
+	}
```

Comment:
We should be able to reuse `wp_scripts_get_suffix()` here as seen  in `wp_default_scripts()` in core:

```suggestion
	$detection_script_file = sprintf( 'detect%s.js', wp_scripts_get_suffix() );
```